Jenkinsfile

```diff
@@ -327,13 +327,14 @@ pipeline {
                 sh '''
                     mkdir -p $WORKSPACE/src/github.com/percona
                     ln -s $WORKSPACE $WORKSPACE/src/github.com/percona/percona-xtradb-cluster-operator
+                    sudo git config --global --add safe.directory /go/src/github.com/percona/percona-xtradb-cluster-operator
```

Comment:
do we really need this? 

I fixed it in PSMDB operator using https://github.com/percona/percona-server-mongodb-operator/pull/1112/commits/0199dd2f91afc97e1aeceba21414a4ef8d397eb5